src/styles/countries.css

```diff
@@ -1,3 +1,37 @@
+.country_card_container {
+  background-color: rgb(165, 14, 99);
+  transition: all 0.3s;
+  margin-bottom: 0.5rem;
+  margin-top: 2rem;
+  min-width: 39rem;
+}
+
+.country_card_container:hover {
+  transition: all 0.4s;
+  transform: scale(1.15);
+}
+
+.grid {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 30%;
+  justify-content: space-between;
+  margin: 5px auto;
+  gap: 0.5rem;
+  text-align: center;
+  padding: 2rem;
+}
+
+.flag__section {
+  height: 250px;
+}
```

Comment:
- [ ] As requested by the previous reviewer, The countries are split into two columns. Please make sure to implement this style for all mobile screens, Currently it's still showing one column.

![image](https://user-images.githubusercontent.com/83803414/207138817-d77ae226-5564-4327-b455-c98e83ab0638.png)